### PR TITLE
Add missing $GI_TYPELIB_PATH to Pango module. Everything that uses GObject-Introspection should have this set.

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/e78780ca2ca17669306d458598b4ea7c for a full test report.

**Comment by migueldiascosta** [comment]
@akesandgren interestingly, 3 of the older easyconfigs have it, but many more don't, can you also add it to those?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 15 out of 17 (17 easyconfigs in this PR)
node2079.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7d04e291d656b240f46d6c9394712aa5 for a full test report.

File: easybuild/easyconfigs/p/Pango/Pango-1.40.1-foss-2016a.eb [hunk 1]
```diff
@@ -34,7 +34,14 @@ builddependencies = [
 
 configopts = "--disable-silent-rules --enable-introspection=yes --enable-static --enable-shared "
 
+sanity_check_paths = {
+    'files': ['bin/pango-view', 'lib/libpango-1.0.%s' % SHLIB_EXT, 'lib/libpangocairo-1.0.%s' % SHLIB_EXT,
+              'lib/libpangoft2-1.0.%s' % SHLIB_EXT, 'lib/libpangoxft-1.0.%s' % SHLIB_EXT],
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pango/Pango-1.40.1-foss-2016a.eb]: @akesandgren There's apparently no `lib/libpangoxft-1.0.so` in `Pango` 1.40.1, so please drop last entry?

File: easybuild/easyconfigs/p/Pango/Pango-1.40.1-intel-2016a.eb [hunk 2]
```diff
@@ -34,7 +34,14 @@ builddependencies = [
 
 configopts = "--disable-silent-rules --enable-introspection=yes --enable-static --enable-shared "
 
+sanity_check_paths = {
+    'files': ['bin/pango-view', 'lib/libpango-1.0.%s' % SHLIB_EXT, 'lib/libpangocairo-1.0.%s' % SHLIB_EXT,
+              'lib/libpangoft2-1.0.%s' % SHLIB_EXT, 'lib/libpangoxft-1.0.%s' % SHLIB_EXT],
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pango/Pango-1.40.1-intel-2016a.eb]: @akesandgren Same here, no `lib/libpangoxft-1.0.so` in `Pango` 1.40.1, so please drop last entry?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 17 out of 17 (17 easyconfigs in this PR)
node2068.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a7bba1ab08643464c4fea848e40b21f4 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**